nemo_text_processing/inverse_text_normalization/inverse_normalize.py

```diff
@@ -28,6 +29,7 @@ class InverseNormalizer(Normalizer):
 
     Args:
         lang: language specifying the ITN
+        whitelist: path to a file with whitelist replacements
```

Comment:
could you also add the format of the whitelist, e.g. written tab normalized,

Reply:
done